commit 1892ce4cdd9bfb5a6c80fe4fdfd4ffcb59d1b06d
Author: Masanari Iida <standby24x7@gmail.com>
Date:   Sat Dec 23 01:41:21 2017 +0900

    doc: md: Fix a file name to md-fault.c in fault-injection.txt
    
    drivers/md/faulty.c has been renamed to md-faulty.c after
    following commit merged int to the main line.
    
      935fe0983e09f4f7331ebf5ea4ae2124f6e9f9e8 .
    
    But the file name in fault-injection.txt has not been changed.
    Now the actual file name and document are in sync.
    
    Signed-off-by: Masanari Iida <standby24x7@gmail.com>
    Signed-off-by: Jonathan Corbet <corbet@lwn.net>

diff --git a/Documentation/fault-injection/fault-injection.txt b/Documentation/fault-injection/fault-injection.txt
index 918972babcd8..1c0bfbbf7869 100644
--- a/Documentation/fault-injection/fault-injection.txt
+++ b/Documentation/fault-injection/fault-injection.txt
@@ -1,7 +1,7 @@
 Fault injection capabilities infrastructure
 ===========================================
 
-See also drivers/md/faulty.c and "every_nth" module option for scsi_debug.
+See also drivers/md/md-faulty.c and "every_nth" module option for scsi_debug.
 
 
 Available fault injection capabilities